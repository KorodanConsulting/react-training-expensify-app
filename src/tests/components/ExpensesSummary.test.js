import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';


test('should redner ExpensesSummary correctly with 1 expense', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={2350} />);
	expect(wrapper).toMatchSnapshot();
}); 

test('should redner ExpensesSummary correctly with multiple expenses', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={235453444350} />);
	expect(wrapper).toMatchSnapshot();
});