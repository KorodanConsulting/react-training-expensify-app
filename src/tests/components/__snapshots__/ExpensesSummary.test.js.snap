// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should redner ExpensesSummary correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $23.50
  </h1>
</div>
`;

exports[`should redner ExpensesSummary correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $2,354,534,443.50
  </h1>
</div>
`;
